clang/runtime/dpct-rt/include/dpct/math.hpp

```diff
@@ -750,6 +779,24 @@ inline unsigned vectorized_sum_abs_diff(unsigned a, unsigned b) {
   return sum;
 }
 
+/// TODO:.
+template <typename VecT, class BinaryOperation1, class BinaryOperation2>
```

Comment:
I think we should all use `class` or `typename`.

Reply:
done